frontend-opal/components/Header/styles.js

```diff
@@ -0,0 +1,88 @@
+import { StyleSheet } from 'react-native';
+
+let displayBorder = (width = 2, style = "solid", color = "black") => (
+    {
+        borderWidth: width,
+        borderStyle: style,
+        borderColor: color,
+    }
+)
+
+export default StyleSheet.create({
+/* ------------------------ Search Area ------------------------ 
+    - a list icon, 
+    - an search input
+    - a search button
+------------------------------------------------------------------ */
+    search: {
+        // Display & Box Model
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-around",
+        // ...displayBorder(3, "solid", "#74b9ff"),
+        width: "100%",
+        height: 50,
+        alignContent: "center",
+
+        // Other
+        backgroundColor: "#0984e3"
+    },
+    listIcon: {
+        // Position
+        marginTop: 14,
+        marginLeft: 20,
+
+        // Display & Box Model
+        // ...displayBorder(2, "solid", "#ff7675"),
+        width: 25,
+        height: 25,
+
+    },
+    input: {
+        // Position
+        marginTop: 10,
+
+        // Display & Box Model
+        // ...displayBorder(2, "solid", "#00b894"),
+        width: "75%",
+        height: 30,
+
+        backgroundColor: "white",
+    },
+    searchButton: {
+        // Position
+        marginTop: 14,
+        marginRight: 20,
+
+        // Display & Box Model
+        // ...displayBorder(2, "solid", "#fdcb6e"),
+        width: 25,
+        height: 25,
+    },
+
+/* ------------------------ Navigation Area ------------------------ 
+    - a portfolio link
+    - a watch list link
```

Comment:
@Yuan-EPAM have the styles speak for themselves instead of this comment, like in the previous section